server/routes/favorites.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const { User } = require("../models/user.models");
+
+//PUT /favorites
+router.put('/',
+async function (req, res, next){
+    const {cityId, add, remove }=req.body;
```

Comment:
we can probably just use `add`. Instead of remove, we can check `if(!add)`

Reply:
got it